src/dune_pkg/toolchain.ml

```diff
@@ -0,0 +1,256 @@
+open! Import
+open! Stdune
+
+module Make = struct
+  let path =
+    lazy
+      (* TODO: prefer gmake over make if it's available for compatibility with the BSDs *)
+      (match Bin.which ~path:(Env_path.path Env.initial) "make" with
+       | Some p -> p
+       | None ->
+         User_error.raise
+           ~hints:[ Pp.text "Install \"make\" with your system package manager." ]
```

Comment:
Nitpick: I think we have a `Pp` formatter for commands.

Reply:
I intentionally left this with regular quotes because it's referring to the package name and not the command.